test/blackbox/docs/AsyncAPI-2_4/streetlights-operation-security.json

```
@@ -0,0 +1,283 @@
+{
+    "asyncapi": "2.4.0",
+    "info": {
+      "title": "Streetlights Kafka API",
+      "version": "1.0.0",
+      "description": "The Smartylighting Streetlights API allows you to remotely manage the city lights.\n\n### Check out its awesome features:\n\n* Turn a specific streetlight on/off 🌃\n* Dim a specific streetlight 😎\n* Receive real-time information about environmental lighting conditions 📈\n",
+      "license": {
+        "name": "Apache 2.0",
+        "url": "https://www.apache.org/licenses/LICENSE-2.0"
+      }
+    },
+    "servers": {
+      "test": {
+        "url": "test.mykafkacluster.org:8092",
+        "protocol": "kafka-secure",
+        "description": "Test broker",
+        "security": [
+          {
+            "saslScram": []
+          }
+        ]
+      },
+      "test_oauth": {
+        "url": "test.mykafkacluster.org:8093",
+        "protocol": "kafka-secure",
+        "description": "Test port for oauth",
+        "security": [
+          {
+            "streetlights_auth": [
+              "streetlights:write",
+              "streetlights:read"
+            ]
+          }
+        ]
+      }
+    },
+    "defaultContentType": "application/json",
+    "channels": {
+      "smartylighting.streetlights.1.0.event.{streetlightId}.lighting.measured": {
+        "description": "The topic on which measured values may be produced and consumed.",
+        "parameters": {
+          "streetlightId": {
+            "$ref": "#/components/parameters/streetlightId"
+          }
+        },
+        "publish": {
+          "summary": "Inform about environmental lighting conditions of a particular streetlight.",
+          "operationId": "receiveLightMeasurement",
+          "traits": [
+            {
+              "$ref": "#/components/operationTraits/kafka"
+            }
+          ],
+          "message": {
+            "$ref": "#/components/messages/lightMeasured"
+          },
+          "security": {
+            "streetlights_auth": [
+              "streetlights:write"
+            ]
+          }
```

The security keyword expects an array 🙂 Take a look here: https://github.com/asyncapi/spec/blob/65d58696f8c3ab6471aa48b12d878221b8d8109e/examples/streetlights-operation-security.yml#L64
```suggestion
          "security": [{
            "streetlights_auth": [
              "streetlights:write"
            ]
          }]
```

You need to apply this to the rest of the channels 🙂